### PR TITLE
Changed attribute loader logic

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * @param options
+ * @returns {Promise<object>}
+ */
+export async function parseI3STileAttribute(arrayBuffer, options) {
+  const {attributeName, attributeType} = options;
+  return {
+    [attributeName]: parseAttribute(attributeType, arrayBuffer)
+  };
+}
+/**
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:15]: empty line

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      return parseStringsAttribute(arrayBuffer);
+    case OBJECT_ID_ATTRIBUTE_TYPE:
+      return parseShortNumberAttribute(arrayBuffer);
+    case FLOAT_64_TYPE:
+      return parseFloatAttribute(arrayBuffer);
+    default:
+      return parseShortNumberAttribute(arrayBuffer);
+  }
+}
+/**
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:33]: ditto

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ * Parse short number attribute.
+ * Short Integer spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.7/attributeStorageInfo.cmn.md
+ * @param arrayBuffer
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:36]: Type the parameters?

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+ * @returns {Uint32Array}
+ */
+function parseShortNumberAttribute(arrayBuffer) {
+  const countOffset = 4;
+  const attributes = new Uint32Array(arrayBuffer, countOffset);
+  return attributes;
+}
+/**
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:44]: ditto

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+ * Parse short number attribute.
+ * Short Integer spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.7/attributeStorageInfo.cmn.md
+ * @param arrayBuffer
+ * @returns {Uint32Array}
+ */
+function parseShortNumberAttribute(arrayBuffer) {
+  const countOffset = 4;
+  const attributes = new Uint32Array(arrayBuffer, countOffset);
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:41]: `return new Uint32Array(...)`?

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+}
+/**
+ * Parse float attribute.
+ * Double Spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.7/attributeStorageInfo.cmn.md
+ * @param arrayBuffer
+ * @returns {Float64Array}
+ */
+function parseFloatAttribute(arrayBuffer) {
+  const countOffset = 8;
+  const arrtibutes = new Float64Array(arrayBuffer, countOffset);
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-attribute.js:52]: typo arrtibutes

**Review by ibgreen — APPROVED** [review]

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js:17]: done

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js:36]: done

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js]: Added

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 4 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js:47]: Fixed

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 5 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js]: done

File: modules/i3s/src/lib/parsers/parse-i3s-attribute.js (hunk 6 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/lib/parsers/parse-i3s-attribute.js]: done

**Pull request merged**